### PR TITLE
1298 acl access control list augmentation

### LINKED ISSUE #1298: ACL: Access Control List - Augmentation
Functionality already exists. Need to understand this and augment it to show the output on startup of which addresses are on the ACL, and then provide the ability to add/remove addresses.

- Control who can and can't make TXs in a network
- Existing functionality: ACL Tool in cdk-erigon - `make acl`
- Show ACL history in a table and make it queryable
	- Possibly an internal API endpoint to query the ACL
- Show last x changes to the ACL on node startup

### PR DESCRIPTION
**Pull request by afa7789**
Adding new features to ACL to make easier to use.

- list of ACL data, to debug.
- README updated with new info.
- added new functions to create a history of transactions in acl.
- at normal acl operations added feature to add transactions to this history table.
- option at acl mode CLI to add new config variable value to the config table.
- added at startup a log of last X acl transactions changes ( 10 being the default ).

### CODE REVIEW

**Comment by afa7789** [comment]
image
image

File: cmd/acl/main.go [hunk 1]
```diff
@@ -7,24 +7,23 @@ import (
...
 	"github.com/ledgerwatch/erigon/params"
-	"github.com/ledgerwatch/erigon/turbo/logging"
-	"github.com/ledgerwatch/log/v3"
+	"github.com/ledgerwatch/erigon/zkevm/log"
 	"github.com/urfave/cli/v2"
 )
 
 func main() {
-	logging.LogVerbosityFlag.Value = log.LvlError.String()
-	logging.LogConsoleVerbosityFlag.Value = log.LvlError.String()
```
Comment (hexoscott) [inline cmd/acl/main.go:20]: could removing these lines be the reason log/v3 isn't working as expected?

**Review by hexoscott — CHANGES_REQUESTED** [review]
Changes look good, but we need to address the logging thing before we can merge

**Comment by hexoscott** [comment]
Hi @afa7789 - are we considering this good to review again?  Did adding the log level stuff back in help with the v3 logging library problem you were seeing?

**Review by revitteth — DISMISSED** [review]

**Review by hexoscott — DISMISSED** [review]

**Review by hexoscott — APPROVED** [review]

**Pull request merged**